core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.ts

```diff
@@ -393,12 +381,14 @@ export class OperatorPropertyEditFrameComponent implements OnInit, OnChanges {
 
   allowChangeOperatorLogic() {
     this.setInteractivity(true);
+    this.workflowActionService.enableWorkflowModification();
```

Comment:
can you move this to `updateWorkflowActionLock` in `ExecuteWorkflowService`, then please revert this whole file

Reply:
done